src/messages/en.ts

```diff
@@ -1,14 +1,6 @@
 import { Messages } from './types';
 
 export const messages: Messages = {
-  // The error messages on required fields
-  required: {
```

Comment:
Do we need to remove this? I usually would say remove things that are unused but after our convo today, I think it would be smart to leave this in and put some comments above it referencing what should and needs to be done in the future. (Talking about the refactor ticket to move all messages into that file instead of inline in the validation.ts file)

Reply:
Good thinking, I've addressed this in a code comment within this file.